tools/virtualization/setting-up-fedora-workstation-domain-on-libvirt.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Setting up Fedora Workstation domain on libvirt  
+subsection: virtualization  
+order: 8  
+---
+
+# Setting up Fedora Workstation domain on libvirt
```

Comment:
This is just a simple `virt-install` + `qemu` usage, right? Could we describe it as such? I think this could be even standalone `virt-install` sub-page.

Reply:
The `virt-install` application conveys setting up of domain to `libvirt` daemon which in turn communicates this with `qemu` so I think the `libvirt` header should be fine. Although, you are right about moving it inside a section specific to `libvirt`.